docs/home/whats-new-2-18.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Version 2.17
+summary: This article describes new features and functionality in the version.
+authors:
+    - Jamie Weider
+date: 2024-June-14
+---
+
+## Release Content - June 30, 2024
+
+* [Deprecation notifications](#deprecation-notifications)
+* [Breaking changes](#breaking-changes)
+
+### Researcher
+
+#### Jobs, Workloads, and Workspaces
```

Comment:
Should we keep using Jobs / Workspaces? or only "Workloads"?

Reply:
@ozRunAI IMHO I think we still need to keep *Jobs* until it is EOL. As long as it still Deprecated, then we need to keep it alive. Another consideration could be that k8s call these jobs. So we need to make clear are w referencing **Run:ai Jobs** or **k8s jobs**.